Base/check-grid.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# check-grid.sh
+
+set -e
+
+# process arguments
+while [[ $# -gt 0 ]]
+do
+    case "$1" in
+        --host)
+        HOST="$2"
```

Comment:
Could you do `HOST="${2:-localhost}"`?

Reply:
Good idea, I was just copying and pasting from other script. Just made the changes and I will push them now. Thanks @tnguyen14